app/src/androidTest/java/com/android/voyageur/ui/e2e/EndToEndTestM2.kt

```diff
@@ -115,6 +115,20 @@ class E2ETestM2 {
         .`when`(userRepository)
         .getUserById(anyString(), anyOrNull(), anyOrNull())
 
+    doAnswer { invocation ->
+          val userIds = invocation.getArgument<List<String>>(0)
```

Comment:
The e2e tests seems flaky, had to do several runs until we got it working. @mirceagavrizi maybe you can take a look a confirm that it's working as intended

Reply:
I agree :)